### PR TITLE
Improve config

### PR DESCRIPTION
**Pull request by ntarocco**

### CODE REVIEW

File: invenio/templates/configurations/uwsgi.yaml [hunk 1]
```diff
@@ -6,31 +6,47 @@ metadata:
...
+    single-interpreter = true
+    need-app = true
+    lazy = true
     lazy-apps = true
-    # thunder-lock = true # https://marc.info/?l=uwsgi&m=140473636200986&w=2
-                          # https://uwsgi-docs.readthedocs.io/en/latest/articles/SerializingAccept.html
+
+    # Silence write errors for misbehaving clients
+    # https://github.com/getsentry/raven-python/issues/732
+    # ignore-sigpipe = true
```
Comment (alejandromumo) [inline invenio/templates/configurations/uwsgi.yaml:33]: Just to be sure, these were added due to `IOError` but are commented. Is that ok?

**Review by alejandromumo — APPROVED** [review]
Thanks a lot! Left a minor comment

File: invenio/templates/configurations/uwsgi.yaml [hunk 2]
```diff
@@ -6,31 +6,47 @@ metadata:
...
+    # ignore-write-errors = true
+    # disable-write-exception = true
+
+    # post-buffering = true
+    buffer-size = 65535
```
Comment (kpsherva) [inline invenio/templates/configurations/uwsgi.yaml:38]: not a blocker, just for my personal knowledge
how were those values calculated? are they standard recommended ?

File: invenio/templates/configurations/uwsgi.yaml (hunk 1 above)
Comment (ntarocco) [reply invenio/templates/configurations/uwsgi.yaml:33]: This is a copy/paste from what has been set in ZenodoRDM OpenShift.
Any settings should be/will be re-checked.

File: invenio/templates/configurations/uwsgi.yaml (hunk 2 above)
Comment (ntarocco) [reply invenio/templates/configurations/uwsgi.yaml:38]: See comment above

File: invenio/templates/deployments/web.yaml [hunk 3]
```diff
@@ -111,28 +111,17 @@ spec:
               name: {{ .valueFrom.secretKeyRef.name }}
               key: {{ .valueFrom.secretKeyRef.key }}
         {{- end }}
-        readinessProbe:
```
[the remark this answers is not in the document]
Comment (ntarocco) [reply invenio/templates/deployments/web.yaml:114]: No, we want :) We can chat IRL

**Pull request merged**